tabu/sampler.py

```diff
@@ -163,7 +175,8 @@ def sample(self, bqm, initial_states=None, initial_states_generator='random',
             # sanity check
             raise ValueError("unknown vartype")
 
-        return dimod.SampleSet.from_samples_bqm((samples, varorder), bqm=bqm)
+        return dimod.SampleSet.from_samples_bqm((samples, varorder), bqm=bqm, num_restarts=restarts)
```

Comment:
Should be keep per-sample info in a sampleset field, or `info`? I can think of pros and cons for both approaches. @hhtong, @arcondello, thoughts?

Reply:
Unless it's a really unusual datatype or something, per-sample info should go in the `SampleSet.record`, that's the whole reason we have it! Especially for stuff like `num_restarts` which is just an int, I have a strong preference for using the record. Will let indexing, truncation, etc to work out of the box.